include/zephyr/drivers/watchdog.h

```diff
@@ -92,6 +92,11 @@ struct wdt_timeout_cfg {
 	struct wdt_window window;
 	/** Timeout callback (can be `NULL`). */
 	wdt_callback_t callback;
+#if defined(CONFIG_WDT_EARLY_WARNING_INTERRUPT_SUPPORT) || defined(__DOXYGEN__)
+	/** Early warning callback (can be `NULL`). */
+	wdt_callback_t ew_callback;
```

Comment:
Do you know if there is any hardware that support both callback to be active?
	wdt_callback_t callback;
	wdt_callback_t ew_callback;

Otherwise we can still keep only one, right?